### ISSUE TITLE
Bundled ffmpeg-for-homebridge markedly slower

### ISSUE THREAD

**Issue by jqnatividad** [Brandawg93/homebridge-nest-cam2#26]
**Describe the bug**
V1.1.9 bundles ffmpeg-for-homebridge.  After upgrading, the performance of the stream got worse and was very laggy using the COPY codec.

**To Reproduce**
Install v1.1.9 and view camera

**Expected behavior**
Better performance than previous releases that didn’t bundle ffmpeg-for-homebridge

Node v10.18.0
HOOBS v3.1.22
Manually compiled ffmpeg 
```
ffmpeg version N-96534-gde1b2aa796 Copyright (c) 2000-2020 the FFmpeg developers
built with gcc 8 (Raspbian 8.3.0-6+rpi1)
configuration: --prefix=/usr/local --arch=armel --target-os=linux --enable-omx-rpi --enable-nonfree --enable-gpl --enable-libfdk-aac --enable-mmal --enable-libx264 --enable-decoder=h264 --enable-network --enable-protocol=tcp --enable-demuxer=rtsp
libavutil      56. 38.100 / 56. 38.100
libavcodec     58. 67.100 / 58. 67.100
libavformat    58. 36.100 / 58. 36.100
libavdevice    58.  9.103 / 58.  9.103
libavfilter     7. 71.100 /  7. 71.100
libswscale      5.  6.100 /  5.  6.100
libswresample   3.  6.100 /  3.  6.100
libpostproc    55.  6.100 / 55.  6.100
```

ffmpeg version in ffmpeg-for-homebridge
```
ffmpeg version 4.2.git-47773f7 Copyright (c) 2000-2020 the FFmpeg developers
built with gcc 6.3.0 (Raspbian 6.3.0-18+rpi1+deb9u1) 20170516
configuration: --pkgconfigdir=/build/workspace/lib/pkgconfig --prefix=/build/workspace --pkg-config-flags=--static --extra-cflags=-I/build/workspace/include --extra-ldflags=-L/build/workspace/lib --extra-libs='-lpthread -lm' --enable-static --disable-debug --disable-shared --disable-ffplay --disable-doc --enable-openssl --enable-gpl --enable-version3 --enable-nonfree --enable-pthreads --enable-libvpx --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libx264 --enable-runtime-cpudetect --enable-libfdk-aac --enable-avfilter --enable-libopencore_amrwb --enable-libopencore_amrnb --enable-filters --enable-decoder=h264 --enable-network --enable-protocol=tcp --enable-demuxer=rtsp --enable-omx-rpi --enable-mmal
libavutil      56. 40.100 / 56. 40.100
libavcodec     58. 68.102 / 58. 68.102
libavformat    58. 38.100 / 58. 38.100
libavdevice    58.  9.103 / 58.  9.103
libavfilter     7. 75.100 /  7. 75.100
libswscale      5.  6.100 /  5.  6.100
libswresample   3.  6.100 /  3.  6.100
libpostproc    55.  6.100 / 55.  6.100
```

**Comment by jqnatividad** [comment]
My workaround was to rename the bundled `ffmpeg` and created a softlink to my manually compiled ffmpeg.

**Comment by Brandawg93** [comment]
I've gotten in contact with the developer of ffmpeg-for-homebridge. I'll update here when I hear back.

**Comment by morilm** [comment]
Hi. Any update ??  I have seen a great performance degradation since the npm ffmpeg is being used...

**Comment by Brandawg93** [comment]
Sorry, everyone. I've been really busy since COVID-19. This should be fixed in [v1.1.11](https://github.com/Brandawg93/homebridge-nest-cam2/releases/tag/v1.1.11). If anyone can try it out and let me know, I'll close this issue.
